docs/sources/datasources/aws-cloudwatch/_index.md

```diff
@@ -10,7 +10,7 @@ weight = 200
 
 Grafana ships with built-in support for CloudWatch. Add it as a data source, then you are ready to build dashboards or use Explore with CloudWatch metrics and CloudWatch Logs.
 
-This topic explains options, variables, querying, and other options specific to this data source. Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+This topic explains options, variables, querying, and other options specific to this data source. Refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
```

Comment:
```suggestion
This topic explains options, variables, and queries that are specific to this data source. For instructions about how to add a data source to Grafana, refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}). Only users with the organization admin role can add data sources.
```
* What is the difference between "options" and "other options".
* Put the what before the reference for consistency.
* "on" means "about" here.

Reply:
I am not sure about "querying" in the midst of those other nouns.

Reply:
Updated.